Base/Python/slicer/util.py

```diff
@@ -2307,7 +2307,10 @@ def infoDisplay(text, windowTitle=None, parent=None, standardButtons=None, **kwa
   """Display popup with a info message.
 
   If there is no main window then the text is only logged (at info level).
+  Skipped during testing.
   """
+  if slicer.app.testingEnabled():
+    return
```

Comment:
To make sure the test effectively covers code that would otherwise be executed, I suggest to move

```python
  if slicer.app.testingEnabled():
    logging.info("Testing mode is enabled: Skipping info popup [%s]" % windowTitle)
    return
```

just above the call to `messageBox` below.

Doing so would also make sure `logging.info(text)` is called.